TimeStepping.pyx

```diff
@@ -67,12 +67,6 @@ cdef class TimeStepping:
         except:
             self.acceleration_factor = 1.0
 
-        try:
-            self.statIOdt = namelist['stats_io']['frequency']
-        except:
-            Pa.root_print('statsIOfrequency set to dt_max')
-            self.statIOdt = self.dt_max
```

Comment:
This is not related to the pressure diagnostic output... Not sure if we could/should delete it